src/address.rs

```diff
@@ -13,6 +13,21 @@ use nom::multi::{many1, separated_list1};
 use nom::sequence::{delimited, pair, separated_pair};
 use nom::{IResult, Parser};
 
+/// A valid OSC method address
```

Comment:
```suggestion
/// A valid OSC method address.
```

Just a missing period at the end.